core/templates/domain/utilities/language-util.service.ts

```diff
@@ -149,25 +149,20 @@ export class LanguageUtilService {
     return language ? language.description : null;
   }
 
-  getContentLanguageDirection(languageCode: string): string {
-    let languageDirection = 'auto'; // Default auto for invalid languageCode
-    this.SUPPORTED_CONTENT_LANGUAGES.forEach((lang: ContentLanguage) => {
-      if (lang.code === languageCode) {
-        languageDirection = lang.direction;
+  getLanguageDirection(languageCode: string): string {
```

Comment:
This is a reasonable simplification, but if you do this it is probably worth ensuring that the two lists are in sync. Perhaps add a frontend test that takes both lists and ensures that there is never a mismatch (i.e. we don't have a language code that says ltr in one list and rtl in the other).

Reply:
I added a test verifying this. I wasn't sure if I approached it the right way, see line 186 in `language-util.service.spec.ts`

Reply:
Yup, what you've done seems reasonable. Thanks!